servant-client/src/Servant/Common/Req.hs

```diff
@@ -71,7 +72,7 @@ data ServantError
   deriving (Show, Typeable)
 
 instance Eq ServantError where
-  FailureResponse a b c == FailureResponse x y z =
+  FailureResponse _ a b c == FailureResponse _ x y z =
```

Comment:
Hmm, `Req` contains a `RequestBody` which doesn't have an `Eq` instance...

Also, I'm not sure one wants to include the request in the failure response equality check -- two responses can be equal even if they didn't originate from the same request.